sale_order_line_price_history/README.rst

```diff
@@ -0,0 +1,101 @@
+=============================
+Sale order line price history
+=============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/11.0/sale_order_line_price_history
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-11-0/sale-workflow-11-0-sale_order_line_price_history
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of Sales to allow you to see the price
+history of a product from a sale order line and set one of this old prices in
+the sale order line.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Sales -> Quotations* and select a Quotation.
+#. Click on the new clock button in one of the sale order lines.
+#. A pop-up will open and you will see the *price history* for the product of
+   the sale order line and for the partner of the sale order.
+#. You can select other partner or left empty to see the price history for all
+   partners.
+#. You can also set the price of one of the price history lines in the sale
+   order line. To do that, you have to open the price history line desired and
+   click the smart button named *Set*.
+
+Known issues / Roadmap
+======================
+
+* The *Customer* column of the *price history lines* list is not hidden when
+  there is a specific client selected because odoo doesn't allow
```

Comment:
s/a/an

Reply:
In that case it must be *a specific*, because the article goes before a consonant.